builder.toml

```
@@ -0,0 +1,50 @@
+description = "Ubuntu jammy base image with buildpacks for Java, Go, and Procfile"
```

```suggestion
description = "Ubuntu 22.04 Jammy Jellyfish base image with buildpacks for Java, Go, and Procfile"
```
to match https://github.com/paketo-buildpacks/builder-jammy-buildpackless-base/blob/f995d3850c23600654f1da3621fb4089084f2b97/builder.toml#L1